recipes/drmaa/run_test.py

```diff
@@ -0,0 +1,4 @@
+try:
+    import drmaa
```

Comment:
You could do the import test in a `run_test.sh` file where you set the DRMAA_LIBRARY_PATH and then run `python -c "import drmaa"`